playbooks/06 - IRP - Case Management/Alert - [02] Capture Ack SLA (Upon Update).json

```diff
@@ -36,63 +36,38 @@
             },
             "status": null,
             "top": "435",
-            "left": "1175",
+            "left": "125",
             "stepType": "\/api\/3\/workflow_step_types\/0bfed618-0316-11e7-93ae-92361f002671",
             "group": null,
             "uuid": "6eb2842d-d208-467b-ab5a-ee0843037d20"
         },
         {
             "@type": "WorkflowStep",
-            "name": "Check Acknowledge SLA Status",
+            "name": "Check Ack SLA Status",
```

Comment:
Check Ack SLA Status > Check Acknowledge SLA Status

Reply:
Done for this playbook and all other playbooks where applicable